### ISSUE TITLE
docker-compose up -d random-accident fails

### ISSUE THREAD

**Issue by thunfischtoast** [codeformuenster/verkehrsunfaelle#14]
Auf Ubuntu 18.04.2

```
chris@chris-lubuntu ~/w/verkehrsunfaelle> sudo docker-compose up -d random-accident
Building random-accident
Step 1/5 : FROM debian:sid-20190506-slim
 ---> e46a3b919e05
Step 2/5 : RUN apt-get update   && apt-get install -y --no-install-recommends   webhook=2.6.9-1   curl=7.64.0-3   gnupg=2.2.13-2   ca-certificates   && curl https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -   && echo "deb http://apt.postgresql.org/pub/repos/apt/ sid-pgdg main" > /etc/apt/sources.list.d/pgdg.list   && apt-get update   && apt-get install -y --no-install-recommends   postgresql-client-11   && rm -rf /var/lib/apt/lists/*
 ---> Running in 37f5fb180539
Get:1 http://cdn-fastly.deb.debian.org/debian sid InRelease [149 kB]
Get:2 http://cdn-fastly.deb.debian.org/debian sid/main amd64 Packages [8332 kB]
Fetched 8480 kB in 8s (1074 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
E: Version '7.64.0-3' for 'curl' was not found
ERROR: Service 'random-accident' failed to build: The command '/bin/sh -c apt-get update   && apt-get install -y --no-install-recommends   webhook=2.6.9-1   curl=7.64.0-3   gnupg=2.2.13-2   ca-certificates   && curl https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -   && echo "deb http://apt.postgresql.org/pub/repos/apt/ sid-pgdg main" > /etc/apt/sources.list.d/pgdg.list   && apt-get update   && apt-get install -y --no-install-recommends   postgresql-client-11   && rm -rf /var/lib/apt/lists/*' returned a non-zero code: 100
```

**Comment by ubergesundheit** [comment]
should be fixed by 62b6e6d

Please close when you are satisfied.

**Comment by thunfischtoast** [comment]
works now, thanks
